admin/sql/CreateFunctions.sql

```diff
@@ -1079,4 +1079,21 @@ CREATE OR REPLACE FUNCTION track_count_matches_cdtoc(medium, int) RETURNS boolea
 $$ LANGUAGE SQL IMMUTABLE;
 
 COMMIT;
+
+-----------------------------------------------------------------------
+-- edit_note triggers
+-----------------------------------------------------------------------
+
+CREATE OR REPLACE FUNCTION a_insert_edit_note() RETURNS trigger AS $$
+BEGIN
+    INSERT INTO edit_note_recipient (recipient, edit_note) (
+        SELECT edit.editor, NEW.id
+          FROM edit_note
```

Comment:
It doesn't look like the edit_note table is used here, should work if you just select from edit where NEW.edit = edit.id?